hera_sim/tests/test_vis.py

```diff
@@ -386,7 +386,7 @@ def test_comparison_zenith(uvdata2):
     ).simulate()
 
     assert viscpu.shape == healvis.shape
-    np.testing.assert_allclose(viscpu, healvis, rtol=0.05)
+    np.testing.assert_allclose(viscpu.conj(), healvis, rtol=0.05)
```

Comment:
We shouldn't have to take the conj here, I don't think. I'm pretty sure work on #165 will remedy this.

Reply:
I'll leave it as-is for now